util/uploadTestResults.js

```diff
@@ -37,23 +37,23 @@ const uploadTestResults = (env, results, options, done) => {
       // Do something with request error
       return Promise.reject(error);
     });
-
-    // Add a response interceptor
-    axios.interceptors.response.use(function (response) {
-      // Any status code that lie within the range of 2xx cause this function to trigger
-      // Do something with response data
-      Debug.log(`Test Analytics success response ${JSON.stringify(response.data)}`);
-      return response;
-    }, function (error) {
-      if (error.response) {
-        Debug.log(`Test Analytics error response: ${error.response.status} ${error.response.statusText} ${JSON.stringify(error.response.data)}`);
-      } else {
-        Debug.log(`Test Analytics error: ${error.message}`)
-      }
-      return Promise.reject(error);
-    });
   }
 
+  // Add a response interceptor
+  axios.interceptors.response.use(function (response) {
```

Comment:
I think it is important to log the error even when the Debug is disabled. Especially now, we are suppressing the error to not fail the job.